### PR TITLE
Tests without subgraphs + Experiment result generation

### PR DESCRIPTION
**Pull request by DovydasVad**
Current experiment results are saved in results/tables directory. There are two types of result tables:
1) Comparison of different score metrics (avg, nn, kl) and k values (10, 10%, -1, -2) with a fixed database.
     - Generation: `python generate_latex_single_table.py -db <database> -r <r_value> --metric [Recall, Reduction]`
2) Comparison of different subgraph types (no subgraphs, star, diamond)
     - Generation: `python generate_latex_comparison_table.py -r <r_value>`

### CODE REVIEW

File: openke/config/Tester.py [hunk 1]
```diff
@@ -199,6 +205,24 @@ def run_ans_prediction(self, topk, outfile_name, dyntop, mode):
...
 
             test_data.append(record)
+        runtime = time.time() - start_time
+
+        hitsHead = 0
+        hitsTail = 0
+        for data in test_data:
+            print(data['head_predictions_raw']['correctness'])
+            print(data['tail_predictions_raw']['correctness'])
+            if 1 in data['head_predictions_raw']['correctness']:
```
Comment (unmeshvrije) [inline openke/config/Tester.py]: use `head_predictions_fil` and `tail_predictions_fil` to compute the recall because it uses the test data only

File: run_tests.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
+import argparse
+import math
+import pickle
+import subprocess
+
+def parse_args():
```
Comment (unmeshvrije) [inline run_tests.py]: Rename the test file with appropriately long name 
`run_all_tests_with_subgraphs_and_without_subgraphs` or something like that.

File: subgraph_predictor.py [hunk 3]
```diff
@@ -15,14 +15,18 @@
 import kge.model
 import torch.nn.functional as F
 import nanopq
+import os
 
 from util import timer
 
+save_dir = '/var/scratch/dvs254/OpenKE-results/'
```
Comment (unmeshvrije) [inline subgraph_predictor.py]: Try not to use global variables to store directory paths and add an input parameter instead.

File: subgraphs.py [hunk 4]
```diff
@@ -260,7 +258,7 @@ def make_diamond_subgraphs(self, sub_type, adj_list_in, adj_list_out):
                 for adj_entity in adj_list[entity]: # change depending on type
                     ent2 = adj_entity[0]
                     rel2 = adj_entity[1]
-                    if ent1 == ent2 or rel1 == rel2:
+                    if ent1 == ent2 or rel1 >= rel2:
```
Comment (unmeshvrije) [inline subgraphs.py:339]: Check if changing the condition to `ent1 > ent2` or `ent1 < ent2` would further decrease the number of diamond subgraphs

**Review by unmeshvrije — CHANGES_REQUESTED** [review]
Please check the comments
